website/content/installation/_index.md

```diff
@@ -112,3 +112,10 @@ secretGenerator:
 generatorOptions:
  disableNameSuffixHash: true
 ```
+
+## Upgrade Current Version
+To upgrade current version of metallb to new release, always look at [the release notes](https://metallb.universe.tf/release-notes/) to see the changes and action required items. Pay special attention when upgrading major/minor releases.
```

Comment:
Not sure if it's me not being fully awake yet, or find the beginning of the sentence confusing. Would something like this be more clear to you?

```suggestion
When upgrading MetalLB, it is recommended to always look at [the release notes](https://metallb.universe.tf/release-notes/) for the changes and action required items. Pay special attention when upgrading major/minor releases.
```

Reply:
"action required items" should be either "action-required items" or "required action items".  Also "it is recommended to" can be deleted.

Other than that, LGTM.